rakelib/run_rspec.rake

```diff
@@ -0,0 +1,65 @@
+require "coveralls/rake/task"
+require "pathname"
+
+require_relative "task_helpers"
+require_relative "example_type"
+
+include ReactOnRails::TaskHelpers
+
+namespace :run_rspec do
+  desc "Run RSpec for top level only"
```

Comment:
@justin808 actually, now that I'm thinking about, I think this happens automatically since I never bothered to add the generated apps to the linters' ignore/exclude lists